include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1950,6 +1950,55 @@ FUNCTION(DistributedActorInitializeRemote,
          ATTRS(NoUnwind),
          EFFECT(Concurrency))
 
+// TODO: move to distributed?
```

Comment:
Do we really need all of these symbols to be exported? I think you can add a private header to the Distributed or Concurrency (since we have some distributed code in Actor.cpp) and just use them in the module.

Reply:
Another node - we are not planning to use these from IRGen either, right?

Reply:
I don't think we need them exported, right -- I wasn't sure how to do it otherwise tbh.
So just a new header in `_Distributed` that declares the right functions?

Reply:
I don't think we'd need those from IRGen...? if anything when we implement the execute func that'd be in SIL I think, so we might want to use them from there then... but we don't yet so whatever simplest thing will work for us.

Reply:
On the swift side I think you can move definitions from Misc to DistributedActorSystem on C++ side you can keep functions where they are and mark them as `SWIFT_RUNTIME_STDLIB_SPI` instead of `EXPORT` and rollback all of the header changes.

Reply:
Awesome, thank you!